R/config.R

```diff
@@ -1,10 +1,12 @@
 gsconfig <- new.env(parent = emptyenv())
+gsconfig$orignial.par <- par(no.readonly = TRUE)
```

Comment:
I'm also wondering if we want to do a dev.new right before catching the par to truly get R's defaults, or is there some off-chance that someone will have set par assume it's going to persist?